.github/workflows/ci-workflow.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         COLLECTOR_LICENSE: ${{ secrets.COLLECTOR_LICENSE }}
         LASP_LICENSE: ${{ secrets.LASP_LICENSE }}
         LASP_SECURE_LICENSE: ${{ secrets.LASP_SECURE_LICENSE }}
-        FORCE_RUN_TESTS_WITH_SECRETS: ${{ github.repository_owner == 'newrelic' && github.head_ref == 'master' }}
+        FORCE_RUN_TESTS_WITH_SECRETS: ${{ github.repository_owner == 'newrelic' && github.event_name == 'push' }}
```

Comment:
This looks good.  Using the 'push' event_name should force a test run with secrets on a PR and any changes to that PR branch.  👍

Reply:
might be a typo but just in case...

this *will not* force runs with secrets on PR's / forked branches.

it *will* force secrets test runs on branches pushed to the main repo via the `push` event and checking for the `newrelic` owner. in a forked repo, `newrelic` will not be the owner (or we did on purpose in newrelic and will setup the secrets). `push` will not be fired for PR's coming from forked repos until the merge into master which is a push to master.

i should have put all this detail in teh description. i am appropriately judging myself right now.

Reply:
also worth noting it will also force on non-master branches in the main repo, which i think is ideal since secrets are available and it gives us another mechanisms for that full validation.